### DISCUSSION TITLE
Message Attempt `responseStatusCode` values

### DISCUSSION THREAD

**Discussion by dariom** [svix/svix-webhooks#1259] [Q&A] [answered]
Hi, we've recently started using Svix and have a question about the Message Attempt API. The [docs](https://api.svix.com/docs#tag/Message-Attempt/operation/v1.message-attempt.get) state that the response message for a successful request will always include a `responseStatusCode` value (as it's marked "required" in the response schema).

I wanted to understand if this is the case with message attempts that time out. Is there a response status code, and what might that be? Can we be sure that a response status code will always be returned?

**Comment by tasn** [comment, marked as answer]
When it's a non-http error, this field is set to 0.

As a side note: the documented schema is enforced to be correct using multiple mechanisms on the Svix end. So if the field is marked as required, it'll always be there (we also don't break API).
